sections/Carousel/CategoryNavigation.tsx

```diff
@@ -0,0 +1,116 @@
+import { ImageWidget } from "apps/admin/widgets.ts";
+import Slider from "../../components/ui/Slider.tsx"
+import Icon from "../../components/ui/Icon.tsx"
+import Image from "apps/website/components/Image.tsx";
+import { useId } from "../../sdk/useId.ts";
+
+export interface Border {
+    /**
+     * @title Cor de Cima
+     * @format color-input
+     */
+    colorTop?: string;
+    /**
+     * @title Cor de Baixo
+     * @format color-input
+     */
+    colorBottom?: string;
+}
+
+/** @titleBy title */
+export interface Category {
+    /** @description Coloque uma imagem nas proporções (110x110)px */
+    image: ImageWidget;
+    alt: string;
+    /** 
+     * @title Borda
+     * @description Em caso de não por uma cor, a borda será preta 
+     */
+    border?: Border;
+    /** @title Título */
+    title: string;
+    link: string;
+}
+
+export interface Autoplay {
+    /** @title Ativar */
+    activate?: boolean;
+    /**
+     * @title Intervalo autoplay 
+     * @description Tempo (em segundos) para trocar de item no carousel autômaticamente.
```

Comment:
"automaticamente" não tem acento
```suggestion
     * @description Tempo (em segundos) para trocar de item no carousel automaticamente.
```